app/assets/v2/js/base.js

```diff
@@ -439,11 +439,38 @@ const gitcoinUpdates = () => {
 
 };
 
+
+if (document.contxt.chat_access_token && document.contxt.chat_id) {
+  // setup polling check for any updated data
+  // scope our polling function so any potential js crashes won't affect it.
+  (($) => {
+    setInterval(() => {
```

Comment:
not sure we need to do this as we are not an SPA maybe just requesting on load is enough.

Reply:
Was asked to remove it from requesting on load.

Reply:
humm why? not sure that make sense since now is being onload and each X seconds

Reply:
we don't check on load anymore in the context.py, we just setup this poll when the page loads.

Reply:
We could remove the interval but than if someone is sitting on the site for any period of time and is not navigating, like on town square or something, then they won't get a dot until they navigate if we don't poll.

Reply:
refactoring this to check for new notifications on page load and to setup an interval